examples/sqlalchemy/01_auto_eager_load.py

```
@@ -0,0 +1,192 @@
+app = Starlette(
+    debug=True,
+    lifespan=lifespan,
+    middleware=[Middleware(DBSessionMiddleware)],
+)
+app.mount("/", graphql_app)
+
+
+if __name__ == "__main__":
+    import uvicorn
+
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Ruff (0.15.14)

[error] Possible binding to all interfaces

(S104)